DatadogInternal/Sources/Models/RUM/GlobalRUMAttributes.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+public struct GlobalRUMAttributes: Codable, PassthroughAnyCodable {
```

Comment:
would it make sense to write extension for `[AttributeKey: AttributeValue]` instead?

Reply:
Extending standard type would bring much less clarity to the whole concept. `GlobalRUMAttributes` is a shared type (it is defined in `DatadogInternal`), standing for contract between RUM and Crash Reporting. Whenever we step on this type in code, we know it is meant for message bus communication. Same could not be achieved with using standard ubiquitous type.

Also, the type defines two aspects that could not be supported in dictionary:
- `Codable` - so we can code it back and forth (`[String: Endocable]` can't be `Codable`);
- `PassthroughAnyCodable` - to avoid marshalling the type when passing it over message bus ([`String: Encodable]` can't be managed this way as we use it transitively in multiple other types exchanged over MB, e.g. for encoding user attributes in RUM view event);